python/tskit/trees.py

```diff
@@ -2893,6 +2893,20 @@ def sackin_index(self):
         """
         return self._ll_tree.get_sackin_index()
 
+    def num_lineages(self, t):
+        """
+        Returns the number of lineages present in this tree at time ``t``. This
+        is defined as the number of branches in this tree (reachable from roots)
```

Comment:
Do you mean "reachable from roots" or "reachable from samples"? (hm, I guess this is the same because roots are defined to be ones reachable from samples, but someone has to remember this definition?) I guess I'd say
```
number of branches in this tree that are reachable from the samples
```

Reply:
done